.github/workflows/ci.yml

```diff
@@ -0,0 +1,57 @@
+# Find full documentation here https://docs.github.com/en/actions/learn-github-actions/workflow-syntax-for-github-actions
+name: CI
+
+on:
+  pull_request:
+
+  # Manual invocation.
+  workflow_dispatch:
+
+  push:
+    branches:
+      - main
+jobs:
+  CI:
+    runs-on: ubuntu-latest
+    permissions:
+      # required by aws-actions/configure-aws-credentials
+      id-token: write
+      contents: read
+    steps:
+      - uses: actions/checkout@v3
+
+      - uses: aws-actions/configure-aws-credentials@v2
+        with:
+          role-to-assume: ${{ secrets.GU_RIFF_RAFF_ROLE_ARN }}
+          aws-region: eu-west-1
+
+      - uses: actions/setup-node@v3
+        with:
+          node-version-file: '.nvmrc'
+          cache: npm
+          cache-dependency-path: 'package-lock.json'
+
+      - name: Setup Java
+        uses: actions/setup-java@v3
+        with:
+          java-version: '8'
+          distribution: 'corretto'
+
+      - name: Install NPM dependencies
+        run: npm ci
+
+      - name: SBT
+        run: sbt "clean;compile;test;Debian / packageBin"
+
+      # Make the filename produced by SBT easier.
+      - name: Rename debian artifact
+        run: mv target/tag-manager_1.0_all.deb target/tag-manager.deb
+
+      - uses: guardian/actions-riff-raff@v2
+        with:
+          projectName: flexible::tag-manager
```

Comment:
What should I use for `projectName`? I took a guess at `flexible::tag-manager` but perhaps...

```suggestion
          projectName: editorial-tools:tag-manager
```

...is better? Given there seem to be two tag-managers in riffraff now.

![Screenshot 2023-09-29 at 16 09 10](https://github.com/guardian/tagmanager/assets/7423751/1e9e982b-9162-4fe2-bfc6-fadb5d4609be)

Reply:
I think `editorial-tools:tag-manager` is the pre-existing one (i.e. the name that's been previously used), so for continuity it might be best to use that?

Reply:
Makes sense, I've made the change 👍